app/templates/application.hbs

```diff
@@ -1,2 +1,3 @@
 <TopNav />
-<SubNav />
\ No newline at end of file
+<SubNav />
+<SideMenu />
```

Comment:
honestly, i think we may want different sidemenu per page, so we can accomplish this one of two ways... before we discuss that though, i believe we need to move this component inside Each page(ex. <SideMenu /> would sit inside jobs/Template.hbs file) 
1.) make this SideMenu component render dynamically by adding props that we pass into it per page
2.) make this SideMenu named specific to each page and only show data specific to its page